### PR TITLE
Add Git Tagger & Update Folders

### PR DESCRIPTION
**Pull request by michaelwehrley**
* Add git_tagger
* Clean up Changelog
* Fix naming folder structure for releases.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -30,6 +30,10 @@ gem "sentry-raven",
     git: "https://github.com/getsentry/raven-ruby.git",
     tag: "0.12.3"
 
+gem 'git_tagger',
+    tag: "1.1.8",
+    git: "https://github.com/eschlange/git_tagger.git"
```
Comment (ericcf) [inline Gemfile]: Mind changing this to the `cbitstech` fork of `git_tagger`?

Comment (michaelwehrley) [inline Gemfile]: :+1:

File: lib/flu-app.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-require "flu-app/version"
+require "flu_app/version"
```
Comment (ericcf) [inline lib/flu-app.rb]: Is this correct? It looks like the folder is `flu-app`.

Comment (michaelwehrley) [inline lib/flu-app.rb]: The convention for git_tagger is to use underscore.

**Comment by ericcf** [comment]
:jack_o_lantern:

**Pull request merged**